hkpy/hkbase/hkobserverfactory.py

```diff
@@ -0,0 +1,111 @@
+###
+# Copyright (c) 2019-present, IBM Research
+# Licensed under The MIT License [see LICENSE for details]
+###
+
+import inspect
+import logging
+import os
+from os.path import dirname, isfile, join
+import traceback
+import glob
+import requests
+import importlib.machinery
+
+from hkpy.hkbase.observer.clients.observerclient import HKBase
+
```

Comment:
Add `__all__` to restrict what other packages can import via `*`. This is important because of line #12 in `hkpy/hkbase/__init__.py`.

Reply:
done